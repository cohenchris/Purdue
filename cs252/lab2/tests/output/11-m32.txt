0x57974f90
(nil)
[
	addr: 3984
	size: 96
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x57975f18
(nil)
[
	addr: 7960
	size: 216
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
0x57976ff4
0x57975f18
[
	addr: 12276
	size: 96
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: 7960
]

[
	addr: 7960
	size: 216
	left_size: 3968
	allocated: false
	prev: 12276
	next: NULLPTR
]

