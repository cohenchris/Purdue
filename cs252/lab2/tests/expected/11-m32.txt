
(nil)
[
	addr: 3984
	size: 96
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
(nil)
[
	addr: 7960
	size: 216
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

BREAK
[
	addr: 12276
	size: 96
	left_size: 3968
	allocated: false
	prev: NULLPTR
	next: 7960
]

[
	addr: 7960
	size: 216
	left_size: 3968
	allocated: false
	prev: 12276
	next: NULLPTR
]

